src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java

```diff
@@ -123,6 +123,64 @@ public void clickOpzioneRevoca() {
         this.revocaDelegaOption.click();
     }
 
+    public void clickMenuPerRifiuto(String nome, String cognome) {
+        try{
+            By menuDelegheBy = By.xpath("//div[@data-testid='delegators-wrapper']//td[@role='cell' and div/p[contains(text(),'"+nome+" "+cognome+"')]]/following-sibling::td[@role='cell']//button[@data-testid='delegationMenuIcon']");
+            getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(menuDelegheBy));
+            this.element(menuDelegheBy).click();
+            logger.info("Si clicca correttamente sul menu della delega");
+        }catch (TimeoutException e){
+            logger.error("Non si clicca correttamente sul menu della delega con errore:"+e.getMessage());
+            Assert.fail("Non si clicca correttamente sul menu della delega con errore:"+e.getMessage());
+        }
+    }
+
+    public void clickRifiuta() {
+        try{
+            By rifiutaButtonBy = By.xpath("//li[contains(text(),'Rifiuta')]");
+            getWebDriverWait(30).until(ExpectedConditions.visibilityOfElementLocated(rifiutaButtonBy));
+            this.element(rifiutaButtonBy).click();
+            logger.info("Si visualizza il bottone rifiuta");
+        }catch(TimeoutException e){
+            logger.error("Non si clicca correttamente sul bottone rifiuta con errore"+e.getMessage());
```

Comment:
insert : after string for divide string and message

Reply:
done